pkgs/top-level/perl-packages.nix

```
@@ -1674,6 +1674,21 @@ let
     buildInputs = [ TestWarn ];
   };
 
+  CatalystAuthenticationStoreLDAP = buildPerlPackage {
+    pname = "Catalyst-Authentication-Store-LDAP";
+    version = "1.016";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/I/IL/ILMARI/Catalyst-Authentication-Store-LDAP-1.016.tar.gz";
+      sha256 = "0cm399vxqqf05cjgs1j5v3sk4qc6nmws5nfhf52qvpbwc4m82mq8";
+    };
+    propagatedBuildInputs = [ NetLDAP CatalystPluginAuthentication ClassAccessorFast ];
+    buildInputs = [ TestMore TestMockObject TestException NetLDAPServerTest ];
```

```suggestion
    buildInputs = [ TestMockObject TestException NetLDAPServerTest ];
```

`NetLDAPServerTest` includes `TestMore`